src/material-experimental/mdc-tabs/tab-group.html

```diff
@@ -52,16 +53,17 @@
   [class._mat-animation-noopable]="_animationMode === 'NoopAnimations'"
   #tabBodyWrapper>
   <mat-tab-body role="tabpanel"
-               *ngFor="let tab of _tabs; let i = index"
-               [id]="_getTabContentId(i)"
-               [attr.tabindex]="(contentTabIndex != null && selectedIndex === i) ? contentTabIndex : null"
-               [attr.aria-labelledby]="_getTabLabelId(i)"
-               [class.mat-mdc-tab-body-active]="selectedIndex === i"
-               [content]="tab.content!"
-               [position]="tab.position!"
-               [origin]="tab.origin"
-               [animationDuration]="animationDuration"
-               (_onCentered)="_removeTabBodyWrapperHeight()"
-               (_onCentering)="_setTabBodyWrapperHeight($event)">
+                *ngFor="let tab of _tabs; let i = index"
```

Comment:
Let's revert all this extra indentation. It makes it hard to rebase older PRs that might add another attribute to the same element.